sdk/textanalytics/ai-text-analytics/src/azureKeyCredentialPolicy.ts

```diff
@@ -1,13 +1,16 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/* eslint-disable @azure/azure-sdk/ts-use-interface-parameters */
```

Comment:
this isn't a big deal, but I'm curious which discrete type is breaking this? If it's the options, you can use `RequestPolicyOptionsLike` instead